lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -76,7 +81,9 @@ sub repair_database {
     # remove broken database
     if (my $err = $@) {
         $log->error("Purging cache directory because database has been corrupted: $err");
-        $db_file->remove;
+        map { $_->remove } ($db_file, $db_file->sibling('cache.sqlite-shm'), $db_file->sibling('cache.sqlite-wal'));
```

Comment:
I personally think using `map` without using the result is bad practice. Just use a for loop here, we're not mapping as we're not mapping anything.

```suggestion
        $_->remove for ($db_file, $db_file->sibling('cache.sqlite-shm'), $db_file->sibling('cache.sqlite-wal'));
```